covid19/spreadsheet_validator.py

```diff
@@ -75,11 +56,11 @@ def format_spreadsheet_rows_as_dict(rows_table, date, state):
         confirmed = getattr(entry, confirmed_attr, None)
         deaths = getattr(entry, deaths_attr, None)
 
-        if not confirmed and deaths or not deaths and confirmed:
+        if (not confirmed and deaths) or (not deaths and confirmed):
```

Comment:
Verificar aqui sobre o `None` vs `0` (usar `v is None`/`v is not None` em vez de `not v`).

Reply:
Resolvido em b2bf207